src/entries/popup/components/CommandK/utils.ts

```diff
@@ -185,7 +185,7 @@ const memoize = (
       commandListVersion = currentVersion;
     }
 
-    if (!resultsCache.has(query)) {
+    if (!resultsCache.has(query) || commandListVersion === currentVersion) {
```

Comment:
Got this Cmd+K cache bug, so had to fix it in here

https://github.com/rainbow-me/browser-extension/assets/53529533/3c5d8a88-ee3b-487a-94b2-8facaf8e97ff

Reply:
It didn't sometimes allow me to search for a token